pbf/views.py

```diff
@@ -50,7 +50,16 @@ def new_game(request):
     game.major_deck.set(Card.objects.filter(type=Card.MAJOR))
     return redirect(reverse('view_game', args=[game.id]))
 
-spirit_starting_energy = {
+# if a spirit starts the game w/more than 0 energy, note that here
+spirit_setup_energy = {
```

Comment:
may I also mention Spreading Hostility and Sunshine

Reply:
Yes, thank you! I audited all of the spirit boards, but not the aspects.